docs/administration/authentication/octopusid-authentication.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Octopus ID Authentication
+description: Octopus Deploy can use Octopus accounts to identify users.
+position: 7
+version: 2019.?.????
```

Comment:
Should this be a real version, or are the `?`s intentional?

Reply:
We don't really use version any more (the functionality still exists, but that's more for partials than anything else). I think we can remove the version from the YAML, and when/if there's a need to specify the version we can do so in the body of the text.

Reply:
Ah, thanks, I'll remove it then